lib/owdit.js

```diff
@@ -2,9 +2,46 @@ const exec = require("executive");
 const formatter = require("./formatters/default")
 const Client = require("node-rest-client").Client;
 const util = require("util");
+const fs = require("fs");
+const path = require("path");
+const rcFilename = ".owditrc";
+
+// Read from .owditrc the lists of packages that should be excluded or only generate warnings
+function readWarnsAndExcludes(cwd) {
+  let warns = [];
+  let excludes = [];
+  const rcFile = path.resolve(cwd, rcFilename);
```

Comment:
Doesn't that mean `.owditrc` needs to be a module (with exports)? I wanted to keep that file as simple as possible, hence plain JSON.